pxr/usd/usdGeom/schema.usda

```diff
@@ -2297,6 +2297,42 @@ class Camera "Camera" (
     However, it follows that if even one property is authored in the correct
     scene units, then they all must be.
 
+    \\section UsdGeom_CameraExposure Camera Exposure Model
```

Comment:
It would be great to rebase this CL on top of dev; now that the exposure schema has landed, it would make the PR much smaller.

Reply:
Done!